### PR TITLE
fix: Pivot focus outline is visible on link-as-tab variant

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

The focus outline on the selected tab was not adequately distinguishable from the background (dark grey on dark blue):
Screenshot of a tab with a dark blue background, white text, and an almost invisible dark border around the blue background

## New Behavior

The focus style for the selected link-as-tab is now an inset white outline:
Screenshot of the same blue tab with a 1px white outline inset by 1px

## Related Issue(s)

- Fixes [16120](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/16120)

### CODE REVIEW

File: packages/react/src/components/Pivot/Pivot.styles.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ const getLinkStyles = (
                   backgroundColor: semanticColors.primaryButtonBackgroundHovered,
                   color: semanticColors.primaryButtonText,
                 },
-                '&:active': {
+                ':active': {
```
Comment (smhigley) [inline packages/react/src/components/Pivot/Pivot.styles.ts:148]: This was a semi-unrelated fix -- using `&` doesn't work inside nested blocks, and is unnecessary in this selector

File: change/@fluentui-react-be1fc47e-c473-4abf-98b8-38adda0aab8b.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: Links in tab style shows focus outline on selected tab",
```
Comment (khmakoto) [inline change/@fluentui-react-be1fc47e-c473-4abf-98b8-38adda0aab8b.json:3]:
```suggestion
  "comment": "fix: Links in tab style show focus outline on selected tab",
```

**Review by khmakoto — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]

**Review by behowell — APPROVED** [review]
